### DISCUSSION TITLE
How do I redirect the https version of the root domain?

### DISCUSSION THREAD

**Discussion by chancellorjax** [YOURLS/YOURLS#3581] [Q&A] [answered]
How do I redirect https://sho.rt (example) to google.com?

I currently have a index.php script with the following code:

```
<?php
header("HTTP/1.1 302 Moved Temporarily");
header("Location: http://google.com");
?>
```

This only redirects the **http** root domain, but not the **https**. How do I redirect both the secure and regular http?

**Comment by dgw** [comment]
Ensure both **http** and **https** handlers in your webserver use the same document root & index file configuration.

You'll have to be more specific about what's going wrong with the **https** redirect if that doesn't help.

BTW, using `header("Location: <whatever>");` already uses HTTP 302 by default, so you don't need the first line. You can set the response code manually with the third argument, e.g. `header("Location: <some_url>", true, 302);`

**Reply by chancellorjax** [reply, marked as answer]
Thanks for the quick reply! Turns out my short domain didn't have a SSL cert. Once I added that, it worked as intended!
